tjctf-2016/README.md

```diff
@@ -0,0 +1,43 @@
+# TJCTF 2016 write-ups
+
+* Scoreboard not released
+
+## Repo-local write-ups
+
+* none
+
+## External write-ups only
+
+* none
+
+## Completed write-ups
+* a-hundred-captchas-60
+* cipher-font-50
+* doge1-30
+* doge2-35
+* gravity-mud-90
+* may-1-2600-100
+* r-u-insane-5
+* reversing-20
+* super-questionable-luggage-service-30
+* znzarmt-mvd-hproo-10
+
+## Missing write-ups
+* basic-images-65
+* contingency-120
+* curses-and-hex-es-65
+* financial-transaction-60
+* flag-seeker-90
+* guess-120
+* interesting-telegrams-100
+* java-sandbox-100
+* mu-sic-200
+* oneshot-170
+* one-time-subtraction-20
+* rsa1-60
+* rsa2-90
+* sketch-airlines-100
+* time-travel-90
+* torrent-90
+* use-the-force-80
+* what-am-i-to-you-90
```

Comment:
looks like you haven't used the writeup tools https://github.com/ctfs/write-ups-tools so the format looks little different

it's okay for now as something is better than nothing, next time use the tools